### PR TITLE
Receive messages in wchat script successfully in terminal chat whole chat displayed automatically.

### PR DESCRIPTION
**Pull request by SinghHimadri**
## Issue that this pull request solves
#77 
Closes: # (issue number)
#77 
## Proposed changes

Brief description of what is fixed or changed
A function getmessage added which fetches the last message sent by our target.Using Command python3 -m wplay -wc <target> now not only messages are sent but the received messages can be viewed.Now the received messages are displayed automatically after every message sent the received message is displayed.No need to switch or any keyboard interrupt it displays the whole chat.
## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface
WhatsApp Image 2020-05-06 at 12 54 22 PM

Screenshot from 2020-05-06 12-54-02

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: wplay/terminal_chat.py [hunk 1]
```diff
@@ -46,10 +46,38 @@ async def chat(target):
...
+        last_message_time = last_message_time[0].replace('[', '')
+        lastMessage = lastMessage.replace(last_message_time, '')
+    except Exception as e:
+        print(e)
+        lastMessage = ""
+    lastOutgoingMessage = ''
+    if tg.lower() in last_message_sender.lower() and lastOutgoingMessage!=lastMessage:
+        print(Fore.GREEN + f"{tg}-", end="")
+        print(lastMessage, end="")
+        print(Style.RESET_ALL)
```
Comment (rpotter12) [inline wplay/terminal_chat.py:82]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5475412)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- wplay/terminal_chat.py  3
         

Clones removed
==============
+ wplay/terminal_chat.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5475412&bid=17943242)

File: wplay/terminal_chat.py [hunk 2]
```diff
@@ -46,10 +46,38 @@ async def chat(target):
...
             message = io.ask_user_for_message_breakline_mode()
+        if 'switch' in message or 'Switch' in message or 'SWITCH' in message :
+            await getMessages(page, target)
 
         # File Share:
         if '#_FILE' in message:
             message.remove('#_FILE')
             await io.send_file(page)
-
+
```
Comment (rpotter12) [inline wplay/terminal_chat.py:56]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5475412)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- wplay/terminal_chat.py  3
         

Clones removed
==============
+ wplay/terminal_chat.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5475412&bid=17943242)

**Comment by SinghHimadri** [comment]
@rpotter12 @xandao6 Please review my PR.

**Comment by SinghHimadri** [comment]
> @SinghHimadri you did this well but it receives message only once like when we start the script it shows the last message. It does not shows the other receive messages when we start chatting.

I have mentioned to see other received messages user must type 'switch' and send this. I have attached screenshot. The target must send a message to view. Everytime we switch we can see the received message.

File: wplay/terminal_chat.py [hunk 3]
```diff
@@ -51,5 +50,30 @@ async def chat(target):
...
+        last_message_time = last_message_time[0].replace('[', '')
+        lastMessage = lastMessage.replace(last_message_time, '')
+    except Exception as e:
+        print(e)
+        lastMessage = ""
+    lastOutgoingMessage = ''
+    if tg.lower() in last_message_sender.lower() and lastOutgoingMessage!=lastMessage:
+        print(Fore.GREEN + f"{tg}-", end="")
+        print(lastMessage, end="")
+        print(Style.RESET_ALL)
```
Comment (rpotter12) [inline wplay/terminal_chat.py:78]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5475412)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Clones removed
==============
+ wplay/terminal_chat.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5475412&bid=17943242)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ wplay/terminal_chat.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5475412&bid=17943242)

**Comment by SinghHimadri** [comment]
> @SinghHimadri you did this well but it receives message only once like when we start the script it shows the last message. It does not shows the other receive messages when we start chatting.

Whole chat is displayed including send and received messages automatically.

**Pull request merged**